src/background.js

```diff
@@ -358,18 +357,21 @@ const rollout = {
 const setup = {
   enabled: false,
   async start() {
-    log("Start");
+    const doneFirstRun = await rollout.getSetting("doneFirstRun");
+
+    log("Start: ",  doneFirstRun);
+
     let runAddon = await browser.experiments.preferences.getBoolPref("doh-rollout.enabled", false);
-    if (!runAddon && !this.enabled) {
+    if (!runAddon && !doneFirstRun) {
       log("First run");
     } else if (!runAddon) {
       log("Disabling");
-      this.enabled = false;
+      // this.enabled = false;
```

Comment:
Short circuit the disable state above by checking for alternative local storage pref.